docs/design/job-scale-up-down.md

```diff
@@ -0,0 +1,63 @@
+# Volcano Job scale up and down
+
+@hzxuzhonghu; April 24, 2020
+
+## Motivation
+
+Currently, Volcano does not support Job update. It is not allowed to update the `Job.Spec` on the fly.
+However, users like ModelArts want to dynamically adjust Job's replicas according to the cluster idle resources 
```

Comment:
That's not only for ModelArts; AFAIK, several ML framework already support elastic model.

Reply:
Will read about that for more context